services/movie.py

```diff
@@ -15,18 +16,22 @@
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Comment:
Using `get()` can raise a `DoesNotExist` exception if no `Movie` with the given `id` is found. Consider handling this exception or using `get_object_or_404` if this is part of a view function.